contents/handbook/who-we-build-for.md

```diff
@@ -0,0 +1,46 @@
+| **Criteria** | - 15-500 employees<br />- $100k+/month in revenue _or_ very large number of consumer users<br />- Raised from leading investors<br />- Not yet IPO'ed |
+| **Why they matter?** | - Able to efficiently monetize them<br />- Very quick sales cycle<br />- Act as key opinion leaders for earlier-stage startups/slower moving companies<br />- Strong opinions on what they need - helping us build a better product |
+| **Examples** | PostHog anytime from their Series B to IPO, Supabase, 11Labs |
+
+Each team will focus more or less on different members of the product team. This is detailed on their team pages.
+
+## Our current Persona
+
+During the life of PostHog, we've tried to build different types of tools for different types of people (or, personas). We've found that, as companies get bigger, the type of person that uses a product changes. As an example:
+
+- We built product analytics for engineers at startups
+- As those companies get a little bit bigger, they'll hire Product Managers who will mostly use product analytics. PMs have more complicated requirements for what a product analytics tool needs to do
+- Even bigger companies often have specialised "analytics engineers". These people are the most demanding.
+
+Some of our products will have multiple personas. It's up to each product team to decide who those are, and how to prioritise between them. 
+
+We've not always been successful at building products for personas other than engineers. We're now at a stage where we need to be in order to continue growing. 
+
+### New products
+
+With AI, we think the biggest change that has happened and will happen is: generalists > specialists. Before, products would be built by teams of engineers, PMs, designers, marketeers and data engineers. Now, a single builder, using AI, can do a lot of these jobs, and that'll continue to be more true over time.
+
+Builders, whether they call themselves developers, product engineers or founders, have always been the most excited users of PostHog, and the keenest to use new products. **That means we want to build new products a builder would use**, products like error tracking, logs and LLM analytics.
+
+There are also products that those builders will often be the first to set up and use, but currently hand over to specialists. Think of a data warehouse (handed over to data engineers), experimentation platform (handed over to PMs, growth engineers or analytics engineers) or messaging platform (marketing). We should build these products so that the builder can delay the point at which they need to hire a specialist to use that product. For example, by making it super easy to set up email campaigns (part written by AI) you don't need to hire a dedicated marketing person for a while.
+
+We don't want to build products where the initial persona is someone other than a builder. Think a CRM, customer support product or marketing tools.
```

Comment:
I find this a bit confusing. Builders will need to collect and act on customer feedback, so they might be the first users for a customer support product. They won't need the kind of tools that our support team needs (rotation, integration with status page, seamless email integration) but they still need some tool.

It's not unreal to assume a company with 15 customers (as suggested above) might not have a dedicated support person if it's in a very technical field IMO.

Reply:
I'm confused by this too. As a founder I would definitely love to have some sort of simplistic CRM and ticketing system if it was incredibly easy to setup and had as few levers as possible. (just remember the slack days and how chaotic that was and how nice it would have been to have a simple ticketing system)

Reply:
I think what this means is that the _initial_ persona should not be a non-builder. So like a CRM that focuses on salespeople first and creating complex deal pipelines is not where we should start. 

Right now we're building customer analytics, to surface information to builders about their customers easily. Maybe down the line a CS person comes in and starts using our customer analytics tool and has more advanced needs - if we've nailed our initial builder persona then we can move toward supporting that persona. But we shouldn't _start_ there.

@timgl does this resonate with what you were thinking here? Or did you mean that we shouldn't build any new products that might eventually have a non-eng persona take them over?